src/app/game-state/activity.service.ts

```diff
@@ -1920,6 +1923,71 @@ export class ActivityService {
       unlocked: false,
       skipApprenticeshipLevel: 0
     };
+    
+    this.CombatCultivation = {
+      level: 0,
+      name: ['Combat Cultivation'],
+      activityType: ActivityType.CombatCultivtion,
+      description: ['Focus on the development of your combat skill.'],
+      consequenceDescription: ['Uses 200 Stamina. A very advanced cultivation technique. Make sure you have killed some monster before attempt this, teaches you how to do combat more effectively.'],
```

Comment:
`      consequenceDescription: ['Uses 200 Stamina. A very advanced cultivation technique. Make sure you have killed some monsters before attempt this, teaches you how to do combat more effectively.']`

You may want to consider using the plural of monster here.

Reply:
I have applied the changes in new commit